product_exception/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Product Exception",
+    "summary": "Ensure products are valid against a set of rules",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Quality",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["hparfr"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["stock", "base_exception"],
```

Comment:
```suggestion
    "depends": ["product", "base_exception"],
```
?, not sure, but I don't see in the code why it should depend from stock

Reply:
You're right, it must depend on the product as it's inheriting the views from the module. There's no relation with the stock module.